<launch>
  <group ns="usb_cam0">
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
      <param name="video_device" value="/dev/video0" />
      <param name="image_width" value="640" />
      <param name="image_height" value="480" />
      <param name="pixel_format" value="yuyv" />
      <param name="camera_frame_id" value="usb_cam" />
      <param name="io_method" value="mmap"/>
    
    </node>
  </group>
  <group ns="usb_cam1">
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
      <param name="video_device" value="/dev/video1" />
      <param name="image_width" value="640" />
      <param name="image_height" value="480" />
      <param name="pixel_format" value="yuyv" />
      <param name="camera_frame_id" value="usb_cam" />
      <param name="io_method" value="mmap"/>
      <remap from="output" to="cam_hub"/>
      </node>
  </group>

  <node name="cam_hub" pkg="base" type="base">
    <remap from="input" to="usb_cam1/usb_cam"/>
  </node>
</launch>